### PR TITLE
test(@toss/react): wrote test code for useForceUpdate

### PR DESCRIPTION
**Pull request by rjsdnql123**
## Overview

A test code has been created to validate the behavior of the useForceUpdate

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/react/src/hooks/useForceUpdate.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  return (
+    <div>
+      <button onClick={onClickTestRefUpdate}>{testRef.current}</button>
+    </div>
+  );
+}
+
+describe('forceUpdate를 호출하여 DOM 업데이트 확인', () => {
+  it('forceUpdate함수 실행 시 DOM이 업데이트 되었는가?', () => {
```
Comment (raon0211) [inline packages/react/react/src/hooks/useForceUpdate.spec.tsx:23]:
```suggestion
describe('forceUpdate', () => {
  it('force-rerenders the component', () => {
```

We write test codes in English; what about rewriting them like this?

**Review by raon0211 — APPROVED** [review]
Thanks for your contribution!

**Review by raon0211 — APPROVED** [review]
Thanks for your contribution!

**Review by raon0211 — APPROVED** [review]
Thanks for your contribution!

File: packages/react/react/src/hooks/useForceUpdate.spec.tsx (hunk 1 above)
Comment (rjsdnql123) [reply packages/react/react/src/hooks/useForceUpdate.spec.tsx]: GOOOOOD!!! Thanks : )

**Pull request merged**